test/hotspot/jtreg/serviceability/jvmti/thread/GetCurrentContendedMonitor/contmon01/contmon01.java

```diff
@@ -119,13 +123,19 @@ public static void test(boolean isVirtual) {
         task.letItGo();
 
         while (waitingBarrier) {
-            doSleep();
+            doSleep(10);
         }
         synchronized (lockFld) {
             System.out.println("\nMain thread entered lockFld's monitor"
                     + "\n\tand calling lockFld.notifyAll() to awake auxiliary thread");
             lockFld.notifyAll();
+
             System.out.println("\nCheck #4: verifying a contended monitor of auxiliary thread ...");
+            while (thread.getState() != Thread.State.BLOCKED) {
+                doSleep(10);
+            }
+            // Wait for 2 seconds to allow the virtual thread to be contended.
+            doSleep(2000);
```

Comment:
This looks like the same sort of "solution" I have been objecting to in #1005. I don't think this is any more correct.